datasets/open_targets/pipelines/_images/copy_bq_datasets/script.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import operator
+import os
+import time
+import typing
+
+from google.api_core.exceptions import ResourceExhausted
+from google.cloud import bigquery_datatransfer_v1
+from google.protobuf.timestamp_pb2 import Timestamp
+
+RETRY_DELAY = 10
+
+
+class TimeoutError(Exception):
+    """Raised when the BQ transfer jobs haven't all finished within the allotted time"""
+
+    pass
+
+
+def main(
+    source_project_id: str,
+    target_project_id: str,
+    service_account: str,
+    dataset_name: str,
+    dataset_versions: typing.List[str],
+    timeout: int,
+):
+    client = bigquery_datatransfer_v1.DataTransferServiceClient()
+    transfer_config_prefix = f"{dataset_name}-copy"
+    transfer_configs = client.list_transfer_configs(
+        request=bigquery_datatransfer_v1.types.ListTransferConfigsRequest(
+            parent=f"projects/{target_project_id}"
+        )
+    )
+
+    existing_configs = [
+        config
+        for config in transfer_configs
+        if config.display_name.startswith(transfer_config_prefix)
+    ]
+
+    _running_configs = []
+    for version in dataset_versions:
```

Comment:
Same here. You don't need a loop if you're only iterating through one dataset, i.e. `["platform"]`

Reply:
@adlersantos done